Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationResultDispatchingValidator.cs

```diff
@@ -33,10 +33,17 @@ public BocListValidationResultDispatchingValidator ()
 
     public void DispatchValidationFailures (IBusinessObjectValidationResult validationResult)
     {
-      ArgumentUtility.CheckNotNull("validationResult", validationResult);
+      ArgumentUtility.CheckNotNull(nameof(validationResult), validationResult);
```

Comment:
Can do here, but I really want to start using the nameof syntax ;)

@MichaelKetting  should I create a Task to fix all argument checks to use the nameof syntax? And should this include some sort of update to the templates? Or is this just overkill?

Reply:
yeah, you can use it here. and yes, that would include the update of the template. please create a task. there's re-sharper structural refactoring or a roslyn rewrite. there will be a couple of failures (we have some special names used until now) but I think it's a pretty safe & straight forward update. you want to check with @a-ctor because it will make merge conflicts with the relations to interface branch so we might want to wait till that's integrated (or not, you two need to figure that out). and we need to scedule it so that it's not gonna break the rest of the release timeline because I'm sure it's not just 1h of work. don't know if it's 4h, 8h or several days, though.